### DISCUSSION TITLE
Is there a way to limit visibility to a single GCS bucket?

### DISCUSSION THREAD

**Discussion by THNJ17** [iterate-ch/cyberduck#16523] [Q&A] [answered]
So I'm trying to set up so I'm connecting only to a single GCS bucket. But currently it requires the IAM permission storage.buckets.list, which would show every bucket I have in my GCP project. Is there a process to limit visibility so I can either connect to a single bucket, or limit it to where I or another user can only see the bucket they've been given principal access to?

Any help would be appreciated, thanks!

**Comment by dkocher** [comment, marked as answer]
Duplicates #16315. Set the bucket name in the _Path_ setting of the bookmark and update to the current [snapshot build](https://docs.cyberduck.io/cyberduck/preferences/#snapshot-builds) available.

**Reply by THNJ17** [reply]
Could you specify what the Path is supposed to look like? Is it just the bucket name?
Currently getting error: "failure to read attributes of {bucket name}" Invalid bucket name: {bucket name} Domain: global. Reason: invalid. Bad request. Please contact your web hosting service provider for assistance

Edit: Fixed! Also needed permission storage.buckets.get and to point the web URL to the bucket in the bookmark. ex: https://console.cloud.google.com/storage/browser/{bucket name}?project={project id}
